apps/web/src/screens/Profile/index.tsx

```diff
@@ -14,6 +15,12 @@ const ProfileScreen = () => {
 		appActions.showSendModal();
 	};
 
+	const handleBuy = () => {
+		Linking.openURL(
+			`https://onramp-sandbox.gatefi.com/?merchantId=${GATEFI_MERCHANT_ID}`,
+		);
+	};
+
```

Comment:
@dev-tronghieu are there any configs to automatically attach the user's address to the onramp page?

Reply:
promote `onramp-sandbox.gatefi.com` to .env variable e.g: `GATEFI_ENDPOINT`